tests/integration/CMakeLists.txt

```diff
@@ -210,7 +210,9 @@ endif ()
 if (ENABLE_TOOLS AND NOT ENABLE_HIP)
   add_subdirectory(replay)
 else ()
-  message(STATUS "Disabling replay tests for HIP build.")
+  if (ENABLE_HIP)
```

Comment:
```suggestion
  if (ENABLE_HIP AND ENABLE_TOOLS)
```

Reply:
So the warning only prints if tools were enabled.

Reply:
Oh jeez... isn't there a better way to write that if statement?
```
if (enable_tools)
  if (not enable_hip)
    add the sub dir
  else
    warning
  endif
endif
```

Reply:
Yeah, my brain is not working 😆